docs/materials/osg/ex3-submit-osg.md

```diff
@@ -19,6 +19,13 @@ You will be using the same exact payload as you did in [Grid exercise 1](/materi
 Now to create submit files that will run in the OSG!
 
 1. If not already logged in, `ssh` into `login04.osgconnect.net`
+1. Set the default project associated with your account by running the following command:
```

Comment:
I think that this should make clear(er) that, once set, this default sticks for all job submissions for this user on this submit server until changed again, even if they log out and log back in. (Assuming that is true.)